pcsx2-qt/input-rec/NewInputRecordingDlg.h

```diff
@@ -15,31 +15,38 @@
 
 #pragma once
 
-#include <QtWidgets/QDialog>
+#include "ui_NewInputRecordingDlg.h"
```

Comment:
Nitpick, but `input-rec` as a folder name doesn't match the rest of the directory structure.

Reply:
In what way? because it has a symbol or because its lower-case? I assume because of the symbol because there are folders with all lower-case and mixed-case currently.

Reply:
In Qt, everything is named TitleCase. (e.g. `Settings`).

Reply:
k, i switched mine to be title-case as well then.  Does that mean this directory should be renamed? https://github.com/PCSX2/pcsx2/tree/master/pcsx2-qt/resources

Reply:
Probably should, yeah.